samples/speaker_recognition/independent_identification.go

```diff
@@ -127,7 +123,7 @@ func IndependentIdentification(subscription string, region string, file string)
 	speakerRecognizer, err := speaker.NewSpeakerRecognizerFromConfig(config, identifyAudioConfig)
 	if err != nil {
 		fmt.Println("Got an error: ", err)
-		return nil
+		return

```

Comment:
`return nil, err`?

Reply:
This function does not return a type in its signature, so this produces: speaker_recognition/independent_identification.go:93:3: too many arguments to return
        have (nil, error)
        want ()

when trying to run.